Dockerfile

```diff
@@ -136,6 +136,7 @@ RUN cd /tmp && \
     pip install setuptools -U && \
     pip install mozdevice==3.0.5 && \
     pip install google-cloud-logging && \
+    pip3 install zstandard==0.11.1 && \
```

Comment:
didn't #17  do this already?

Reply:
Yes. I hadn't seen the existing PR, and it hadn't landed when I wrote this.